src/dmd/backend/outbuf.d

```diff
@@ -43,7 +45,10 @@ struct Outbuffer
     @trusted
     void dtor()
     {
-        free(buf);
+        if (length >= huge)
+            free(buf);
+        else
+            free(buf);
```

Comment:
What is the point of this condition if `free(buf)` is called unconditionally? Also, is this related in any way?

Reply:
ouch my work in progress spilled... sorry! fixing now